blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -112,3 +121,56 @@ columns with a value each to identify the partition to alter.
 :value: The columns value.
 
 .. SEEALSO:: :ref:`Alter Partitioned Tables <partitioned_tables_alter>`
+
+
+Sub-commands
+============
+
+``REROUTE``
+-----------
+
+The ``REROUTE`` command provides different options to allocate a shard on a node.
```

Comment:
different... -> various options to manually control the allocation of shards.

Reply:
It allows to enforce explicit allocations, cancel them and makes it possible to move shards between nodes in a cluster.